source/documentation/getting_started/get_started.erb

```diff
@@ -135,6 +139,88 @@ You can practice deploying an app by deploying a test static HTML page.
 
     If you do not specify the app name in `cf push`, the name specified in the manifest file is used.
 
-    The static HTML page is now available at your [app domain](/orgs_spaces_users.html#regions).
+The static HTML page is now available at your [app domain](/orgs_spaces_users.html#regions).
 
 For a production app, you should read the [production checklist](deploying_apps.html#production-checklist).
+
+## Deploy a test static HTML page using Cloud Foundry API v3
+
+You can practice deploying an app by deploying a test static HTML page.
+
+The following instructions tell you how to deploy a static HTML page using [v3 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v3). This version of the API allows you to push an app with no downtime without needing to use a [blue-green deployment](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html).
+
+If you are using [v2 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v2), refer to the documentation on [deploying a test static HTML page using Cloud Foundry API v2](get_started.html#deploy-a-test-static-html-page-using-cloud-foundry-api-v2).
+
+<br>
+
+<%= warning_text('Cloud Foundry API v3 is still in beta.') %>
+
+1. To deploy an app, you must select a [target](deploying_apps.html#set-a-target). This is a combination of an [organisation](/orgs_spaces_users.html#organisations) and a [space](/orgs_spaces_users.html#spaces).
+
+    All orgs have a sandbox space for you to use when learning about the PaaS. When deploying a test static HTML page, you should target this sandbox space by running:
+
+    ```
+    cf target -o ORGNAME -s sandbox
+    ```
+
+    where `ORGNAME` is your org and `sandbox` is the name of the sandbox space.
+
+    If you deploy an app using the same name and target as an existing app, the original will be replaced. If you are not sure about where to deploy your app, consult the rest of your team or speak to the PaaS team by emailing [gov-uk-paas-support@digital.cabinet-office.gov.uk](mailto:gov-uk-paas-support@digital.cabinet-office.gov.uk).
+
+2. In an empty directory, create an `index.html` file containing the following markup:
+
+    ```
+    <html>
+      <head>
+        <title>Static Site</title>
+      </head>
+      <body>
+        <p>Welcome to the static site!</p>
+      </body>
+    </html>
+    ```
+
+3. Create a `manifest.yml` file in the same directory as the `index.html` file. The manifest file tells Cloud Foundry what to do with your app.
+
+    ```
+    ---
+    applications:
+    - name: APPNAME
+      memory: 64M
+      buildpack: staticfile_buildpack
```

Comment:
Technically this style is deprecated now, it should be:

```suggestion
      buildpacks:
      - staticfile_buildpack
```

Reply:
We have a link to https://docs.cloudfoundry.org/buildpacks/#system-buildpacks in the Buildpacks section. @andy-paine are you talking about links to specific individual buildpacks?